cms/settings/formats/en_GB/formats.py

```diff
@@ -0,0 +1,6 @@
+# 1 November 2025
+# as opposed to "j M Y" which is '1 Oct 2024'
+DATE_FORMAT = "j F Y"
+# 1 November 2024, 1p.m.
+# as opposed to "j M Y, P" which is '25 Oct 2006, 2:30 p.m.'
+DATETIME_FORMAT = "j F Y g:ia"
```

Comment:
Minor, this doesn't match the live site exactly; we should check if that is a problem. Ours would be `a.m.` and p.m.` vs live site which is `am` and`pm`

Reply:
This has to do with how the Django date filter work - https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#std-templatefilter-date

If it is a problem, we could look at providing a custom one.

Reply:
As discussed, let keep as-is and revisit if it becomes a problem.